adobe_vipm/flows/fulfillment.py

```diff
@@ -76,20 +77,20 @@ def _create_subscription(mpt_client, seller_country, customer_id, order, item):
         customer_id,
         item["subscriptionId"],
     )
-    order_item = get_order_item(order, item["extLineItemNumber"])
+    order_line = get_order_item(order, item["extLineItemNumber"])
     subscription = {
-        "name": f"Subscription for {order_item['name']}",
+        "name": f"Subscription for {order_line['item']['name']}",
         "parameters": {
             "fulfillment": [
                 {
-                    "name": PARAM_SUBSCRIPTION_ID,
+                    "externalId": PARAM_SUBSCRIPTION_ID,
                     "value": item["subscriptionId"],
                 }
             ]
         },
-        "items": [
+        "lines": [
             {
-                "lineNumber": item["extLineItemNumber"],
+                "id": item["extLineItemNumber"],
```

Comment:
Here item["extLineItemNumber"] is an integer so we should rebuild the MPT line ID.